webapp/config/themeConfig/themeCreator.js

```diff
@@ -366,6 +366,15 @@ const themeCreator = (_themeVariables = {}, _themeConfig = {}) => {
       return style;
     },
 
+    condensedTextView: {
+      copyIcon: {
```

Comment:
Since there is already another copyIcon style object and the generated styles are the same, we could probably generalize this and share the styling `theme.copyIcon` in both spots. This is useful so that anytime someone wants to use or apply the copyIcon it will look the same and have consistent styles across themes. 

To be backwards compatible, the one in the table should probably stay and could just destructure the generalized one.

Reply:
Okay, I agree with your feedback overall.

In the case of implementing themeConfig.copyIcon, we would not be able to reference themeConfig.copyIcon within themeConfig.expandedRow.copyIcon.

I think we need to implement something like the following or replace themeConfig.expandedRow.copyIcon with themeConfig.copyIcon across the organization. Could you elaborate on your suggestion to destructure the generalized one or your thoughts on this proposal?

```
themeConfig {
  ...,
  copyIcon {...},
  expandedRow {
  ...,
```
  ~~~copyIcon~~~
```
  },
  ...
}

themeConfig.expandedRow.copyIcon = themeConfig.copyIcon;
```

![image](https://user-images.githubusercontent.com/18730408/43617318-34a49284-9676-11e8-91f7-47b636384e85.png)

Reply:
Yeah, good call. Forgot that they're in the same object so wouldn't be able to use it 😊. Updating it across the whole organization would certainly be the cleanest. And actually... I don't even think that ExpandedRow.js file is being used anymore since we generalized it and made it a common component in bpanel-ui. I'll do a PR to get rid of that, and then let's just move to `themeConfig.copyIcon` which only also needs to be applied to ExpandedDataRow.js in bpanel-ui. 

Good catches!

Reply:
removed/removing from recent-blocks! https://github.com/bpanel-org/recent-blocks/pull/9